packages/helium-cli/src/utils.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  createAtaAndMintInstructions,
+  createMintInstructions,
+  sendInstructions,
+  toBN
+} from "@helium/spl-utils";
+import {
+  createCreateMetadataAccountV3Instruction,
+  PROGRAM_ID as METADATA_PROGRAM_ID
+} from "@metaplex-foundation/mpl-token-metadata";
+import * as anchor from "@project-serum/anchor";
+import {
+  Connection,
+  Keypair,
+  PublicKey
+} from "@solana/web3.js";
+import fs from "fs";
+import fetch from "node-fetch";
+
+export async function exists(
+  connection: Connection,
+  account: PublicKey
+): Promise<boolean> {
+  return Boolean(await connection.getAccountInfo(account));
+}
+
+export async function createAndMint({
```

Comment:
This function is duplicating a lot of the logic that already exists in spl-utils

Reply:
Not duplicating, it's using the functions from spl utils.